### DISCUSSION TITLE
I get internal server error on pages that i use getserversideprops

### DISCUSSION THREAD

**Discussion by Oladejiraji** [vercel/next.js#55290] [Help]
### Summary

I have an app that has a lot of 3d modelling and I started getting internal server error when i use getserversideprops. I checked the error on the runtime logs and got this error.
Error: /lib64/libz.so.1: version `ZLIB_1.2.9' not found (required by /var/task/node_modules/canvas/build/Release/libpng16.so.16)
I am aware that you can get this error while building, but this is the first time i'm getting it on specific pages.

**Comment by icyJoseph** [comment]
Most likely because when the server side code for `GetServerSideProps` runs, you have a missing dependency, that you may have locally. In other words, some code is being executed when the request arrives to your server. Note that imports might run code at their "top level" and that could be running more code than it is immediately obvious to you.

**Reply by icyJoseph** [reply]
Note that this dependency might be a system library dependency, and you'd probably have to make sure to install it.
